llvm/test/Transforms/LoopVectorize/first-order-recurrence-complex.ll

```diff
@@ -335,7 +335,7 @@ define void @cannot_sink_reduction(i32 %x, ptr %ptr, i64 %tc) {
 
 
 
-; CHECK-NET:     ret void
+; CHECK-NEXT:    ret void
```

Comment:
please remove this line as it has been covered by line 333 (which was probably generated by UTC).